extensions/arc/src/models/miaaModel.ts

```diff
@@ -137,9 +137,9 @@ export class MiaaModel extends ResourceModel {
 		}
 	}
 
-	public async callGetDatabases(): Promise<void> {
+	public async callGetDatabases(isBackupPage: boolean = false): Promise<void> {
```

Comment:
Generally I'd suggest that the model not know anything about the UI - so instead having this be something more generic like `doDryRun`